R/derive_param_extreme_record.R

```diff
@@ -136,6 +136,17 @@ derive_param_extreme_record <- function(dataset = NULL,
                                         order,
                                         mode,
                                         set_values_to) {
+  deprecate_inform(
+    when = "1.2.0",
+    what = "derive_param_extreme_record()",
+    with = "derive_extreme_event()",
+    details = c(
+      x = "This message will turn into a warning for at least one year.",
```

Comment:
I think this message is misleading. If a user calls the function in December 2025, it looks as if the warning will not occur before end of 2026 although we will change it at the beginning of 2026.

What about:
```suggestion
      x = "This message will turn into a warning at the beginning of 2026.",
```